style.css

```diff
@@ -0,0 +1,134 @@
+
+body {
+    background-color: #f5f7fa;
+    padding: 3rem;
+    font-family: 'Teko', sans-serif;
+}
+h1,h2,h3,h4,h5 {
+    margin: 0;
+    padding: 0;
+}
+details{
+    margin-left: 2rem; 
+}
+.container {
+    height: 100%;
+    width: 100%;
+    position: relative;
+    margin: 0 auto;
+    display: flex;
+    flex-wrap: wrap;
+    background-color: #c4b5b5;
+    box-shadow: 2rem 2rem 2rem 0 rgba(0, 0, 0, 0.4);
+    border-radius: 1rem;   
+}
+.container-left {
+    width: 25%;
+    float: left;
+    background-image: linear-gradient( 102.1deg,  rgba(96,221,142,1) 22.7%, rgba(24,138,141,1) 76.1% );
+    border-radius: .6rem;   
+}
+.container-right {
+    height: 100%;
+    width: 70%;
```

Comment:
currently the layout breaks during resizing. Take a look at `box-sizing: border-box`  -that should help